### PR TITLE
Remove 16.10 changewave

### PR DESCRIPTION
**Pull request by Forgind**
### Context
Removes the 16.10 change wave and associated tests and edits documentation.

This is based on the respect deps.json pr to avoid an imminent merge conflict.

### CODE REVIEW

**Review by rainersigwald — COMMENTED** [review]
LGTM after the other thing gets merged + rebase.

**Comment by BenVillalobos** [comment]
First glance looks good, will need to wait until the other PR merges to see a cleaner diff. This was in the back of my mind as of late, thanks for doing this!

**Comment by Forgind** [comment]
> First glance looks good, will need to wait until the other PR merges to see a cleaner diff. This was in the back of my mind as of late, thanks for doing this!

If you want to see a cleaner diff, I made the commits really nice—the last four are the relevant ones.

File: src/Framework/ChangeWaves.cs
Comment (BenVillalobos) [inline src/Framework/ChangeWaves.cs:28]: Do we want 17_2 stuff in here? Latest/vs17.2 doesn't have the wave, and if something gets merged into main now, the first LTS release that will see a new changewave feature is 17.4.

Comment (Forgind) [reply src/Framework/ChangeWaves.cs:28]: I talked with marcpopMSFT about that a few days ago, and we decided yes. There isn't anything in it, but it avoids future confusion.

**Review by BenVillalobos — APPROVED** [review]

**Review by rainersigwald — APPROVED** [review]

**Pull request merged**
